pyproject.toml

```diff
@@ -9,20 +9,19 @@ authors = [
 ]
 
 dependencies = [
-    "invenio-app-rdm[opensearch2] ~=12.0.0",
-    "invenio-assets >=3.1.0,<3.2.0",
-    "invenio-logging[sentry_sdk] ~=2.0",
+    "invenio-app-rdm[opensearch2] ~=13.0.0b3.dev2",
+    "invenio-logging[sentry_sdk]>=4.0.0,<5.0.0",
     "uwsgi >=2.0",
     "uwsgitop >=0.11",
     "uwsgi-tools >=1.1.1",
-    "invenio-override ~=0.0.3"
+    "invenio-override[marc21] ~=0.0.4",
 ]
 
 [tool.setuptools]
 py-modules = []
 
 [tool.uv.sources]
 invenio-override = { git = "https://github.com/sharedRDM/invenio-override", branch = "main" }
-#invenio-override = { path = "/Users/lpandath/Documents/work/rdm/invenio/invenio-instances/invenio-version-9/invenio-override", editable=true }
+#invenio-override = { path = "/home/eduard/Schreibtisch/invenio-override", editable=true }
```

Comment:
*Note* - this is okay for now but should remove this once you have a release from instance v0.13.0.